### ISSUE TITLE
Can Sonar check code from the CLI?

### ISSUE THREAD

**Issue by alistair-hmh** [SonarSource/SonarJS#1108]
I would like to setup SonarJS to run from the CLI. Is this possible?

**Comment by vilchik-elena** [comment]
There is eslint plugin with _some_ SonarJS rules https://github.com/SonarSource/eslint-plugin-sonarjs.
Otherwise, I think there is no more way to get issues as CLI. SonarQube/SonarCloud and/or SonarLint are recommended

**Comment by alistair-hmh** [comment]
Thanks for your quick response! 

Can I check something? 

If I add `1: SonarJS ESLint rules` to my project alongside `2: other ESLint rules`, can I get reports in SonarCube on both the sets of ESLint rules?

**Comment by vilchik-elena** [comment]
Yes, generate json ESLint report and import it in Sonar**Q**ube https://github.com/SonarSource/SonarJS/blob/master/docs/DOC.md#eslint
